doc/source/sdk/v2/migrating.md

```diff
@@ -0,0 +1,124 @@
+## Migrating to v2
+The first step to migrate your legacy app to the Zendesk App Framework (ZAF) v2 is to clone or fork the [Iframe App Template](https://github.com/zendesk/iframe_app_template). This template contains a very simple example app and includes build tools to support some features previously provided by ZAF v1, such as jQuery, Bootstrap 2.3, Handlebars and underscore. It also includes a library of shims for some legacy v1 functions, such as `renderTemplate`, `switchTo` and `setting`. The Iframe App Template also serves as an adaptor for your legacy app, so that your `events` and `requests` definitions can be translated to work with ZAF SDK v2.
+
+### How does it work?
+ZAF v1 had a strict folder structure and very limited support for external libraries and modules. In contrast, ZAF v2 allows you to structure your files and folders in whichever way you like. You are also free to include external libraries and third-party modules. In order to make this possible, ZAF v2 delegates some responsibilities to the app developer. For example, the Iframe App Template uses webpack to compile Handlebars templates, SCSS stylesheets and CommonJS modules. If you'd like to use any other technologies you're welcome to do so, but you may need to re-configure the build process yourself.
```

Comment:
Original:
> ...the Iframe App Template uses webpack to compile Handlebars templates, SCSS stylesheets and CommonJS modules.

Suggested edit: Maybe add a link to webpack page @ http://webpack.github.io/ to explain what it is -
> ... the Iframe App Template uses [webpack](http://webpack.github.io/) to compile Handlebars templates, SCSS stylesheets, and CommonJS modules.